### PR TITLE
[Uno] Fluent alignment

### PR DESCRIPTION
**Pull request by lukeblevins**
- Update to latest Fluent UI symbols font
- Improvements to control styles
- Tweak typography and colors
- Switch to MUX Expander, NavigationView, and InfoBadge controls
- Remove hardcoded splash color
- Remove unused Uno platform projects

Untitled

### CODE REVIEW

**Comment by lukeblevins** [comment]
This pull request replaces my previous one as a focused set of mostly cosmetic changes. There will be more work to do in fixing indentation inconsistencies and testing for potential regressions on newer Uno.UI versions.

File: Uno/NugetPackageExplorer.Legacy/Presentation/ShellViewModel.cs [hunk 1]
```diff
@@ -3,19 +3,33 @@
...
 
-		public ShellViewModel()
-		{
-		}
-	}
+#if __WASM__
+            IsStandaloneWindow = WebAssemblyRuntime.InvokeJS(@"window.matchMedia('(display-mode: standalone)').matches") == "true";
+#else
+            IsStandaloneWindow = true;
+#endif
```
Comment (jeromelaban) [inline Uno/NugetPackageExplorer.Legacy/Presentation/ShellViewModel.cs:32]: This needs to be reevaluated somehow, as when the webapp is being installed as an app, the button does not unless the app is restarted. I wonder if we could mitigate this by determining that we should reevaluate this property when the size of the window changes very significantly (which happens when the app is installed).

File: Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml [hunk 2]
```diff
@@ -1,1017 +1,877 @@
...
+                                                                    Visibility="{Binding Package.Package.PublisherSignature, Converter={StaticResource NullToCollapsed}}"
+                                                                    Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                        <wct:HeaderedContentControl Header="{x:Bind pexpl:Resources.Dialog_RepositoryLabel}"
+                                                                    Content="{Binding Package.Package.RepositorySignature}"
+                                                                    ContentTemplate="{StaticResource RepositorySignatureTemplate}"
+                                                                    Visibility="{Binding Package.Package.RepositorySignature, Converter={StaticResource NullToCollapsed}}"
+                                                                    Style="{StaticResource LabeledTextContentControlStyle2}" />
+                                    </legacyControl:StackPanelEx>
+                                </muxc:Expander>
+                                <muxc:Expander Header="Health"
```
Comment (jeromelaban) [inline Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml:290]: The expander is now rotating its chevron when closed, I wonder if it's an Uno issue.

Comment (lukeblevins) [reply Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml:290]: @jeromelaban I noticed on newer versions of Uno packages the Expander fails to display altogether which is definitely a regression

Comment (jeromelaban) [reply Uno/NuGetPackageExplorer.Shared/InspectPackage.xaml:290]: Got it. Could be related to https://github.com/unoplatform/uno/issues/11596 and https://github.com/unoplatform/uno/issues/9062

**Pull request closed without merging**